python_src/src/lamp_py/migrations/env.py

```diff
@@ -0,0 +1,96 @@
+from logging.config import fileConfig
+
+from alembic import context
+
+from lamp_py.postgres.postgres_utils import get_local_engine
+from lamp_py.runtime_utils.import_env import load_environment
+
+# load local environment, if running alembic locally
+load_environment()
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# gate to make sure alembic is run using -n flag
+if config.config_ini_section == "alembic":
+    raise SyntaxError("Run alembic with -n flag to specifiy Database name.")
+
+# get database name from -n flag when alembic is run from cmd line
+db_name = config.config_ini_section
+
```

Comment:
`db_name` is equivalent to `config_ini_section` this is the valued passed into alembic with "-n" flag.